src/sentry/replays/usecases/ingest/event_parser.py

```diff
@@ -716,6 +741,20 @@ def parse_click_event(payload: dict[str, Any], is_dead: bool, is_rage: bool) ->
     )
 
 
+def parse_multiclick_event(payload: dict[str, Any]) -> MultiClickEvent | None:
+    click_event = parse_click_event(
+        payload,
+        is_dead=False,
+        is_rage=payload["data"].get("clickCount", 0) >= RAGE_CLICK_COUNT_THRESHOLD,
+    )
+    if not click_event:
+        return None
+    return MultiClickEvent(
+        click_count=payload["data"].get("clickCount", 0),
+        **click_event.__dict__,
```

Comment:
It's better to embed the click event directly.  `click_event=click_event`. `click_event: ClickEvent`.  That way you don't need to redefine ClickEvent within the MultiClickEvent type.  _Compose_ it.

Now your definition for MultiClickEvent is one field away from being done instead `n` fields.